app/build.gradle.kts

```diff
@@ -455,6 +456,8 @@ dependencies {
   lintChecks(projects.lint)
 
   runtimeOnly(libs.jackson.core)
+
+  implementation(libs.datadog.sdk)
```

Comment:
Just a personal preference, we can probably add dependencies in alphabetical order. 😅

Reply:
Hmm, I prefer to add new dependencies at the bottom of the file, so it matches the version control history 😅

Reply:
Yeah, I figured to keep this alphabetical to make it easier to find dependencies. But sure, this works. We can refactor it once in a while I guess.